README.md

```diff
@@ -47,7 +47,7 @@ pr: https://github.com/elastic/elastic-agent/pull/823
 Prerequisites:
 - installed [mage](https://github.com/magefile/mage)
 - [Docker](https://docs.docker.com/get-docker/)
-- [X-pack](https://github.com/elastic/beats/tree/main/x-pack) to pre-exist in the parent folder of the local Git repository checkout
+- [beats](https://github.com/elastic/beats) to pre-exist at ../beats
```

Comment:
Actually the `beats` repo is only required if compiling and packaging the beats as well, with `EXTERNAL=false`. Therefore it can also be stated there:

```suggestion
- [beats](https://github.com/elastic/beats) to pre-exist at ../beats if, and only if, packaging with `EXTERNAL=false` to package the beats as well
```

Reply:
@codefromthecrypt would you mind adding what I mentioned above?

Reply:
good idea, done!